packages/@aws-cdk/aws-stepfunctions-tasks/test/codebuild/stop-build.test.ts

```diff
@@ -0,0 +1,62 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { CodeBuildStopBuild } from '../../lib/codebuild/stop-build';
+
+let stack: Stack;
+let buildId: string;
+
+beforeEach(() => {
+  //GIVEN
+  stack = new Stack();
+  buildId = 'CodeBuildId';
```

Comment:
add a test with a `path` as well (to ensure the `Id.$` is rendered as the key in `Paramters` as we would expect.